pylint_odoo/checkers/no_modules.py

```diff
@@ -390,6 +390,12 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def _convert_value_to_name(self, attribute):
+        """Convert 'Field Case SENSITIVE' to 'field_case_sENSITIVE' """
+        parts = [(part[0].lower() + part[1:]) for part in attribute.split(' ')]
```

Comment:
Must be converted from 'Field Case SENSITIVE' to 'field_case_sENSITIVE'

Is not there a function for that?

Already the word comes in capitalize!